File: WebBibliotecaApp/src/Objects/Colecao.ts
export class Colecao {
  codColecao = 0;
  nomeColecao: string = "";
  autor: string = "";
  editora: string = "";
  anoLancamento = 0;

  constructor(CodColecao: number, NomeColecao: string, Autor: string, Editora: string, AnoLancamento: number) {
    this.codColecao = CodColecao;
    this.nomeColecao = NomeColecao;
    this.autor = Autor;
    this.editora = Editora;
    this.anoLancamento = AnoLancamento;
  }
}

File: WebBiblioteca_Api/BibliotecaApiDLL/Colecao/Colecao.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BibliotecaApiDLL.Colecao
{
    public class Colecao
    {
        public Colecao(int codColecao, string nomeColecao, string autor, string editora, int anoLancamento)
        {
            CodColecao = codColecao;
            NomeColecao = nomeColecao;
            Autor = autor;
            Editora = editora;
            AnoLancamento = anoLancamento;
        }

        public int CodColecao { get; set; }
        public string NomeColecao { get; set; }
        public string Autor { get; set; }
        public string Editora { get; set; }
        public int AnoLancamento { get; set; }


        
    }
}
